contents/handbook/company/team/joe-martin.md

```diff
@@ -8,30 +8,54 @@ showTitle: true
-- I'm truly uncomfortable with public praise or attention. A private thanks is better than a spotlight.
-- I care deeply about my co-workers and trust them by default. I'm always happy to chat, about anything.
-- I'm usually fine with ad-hoc meetings, even on no-meeting days, but weekends are sacred to me. Slack is the best way to reach me. I generally work 9-6 GMT, with no break for lunch.
-
-# What I value
-- Compassion and inclusivity. Anything that puts people first, really. 
-- A 'no bad ideas' attitude. I don't think there's any such thing as a stupid question, even though I ask a lot of them.
-- Vulnerability, especially in leaders. It's healthy and builds trust. I'm happy to be vulnerable with you too.    
-- Honest feedback. I am literally always interested in hearing how I can improve.
-- A hands-on attitude. I don't think I've ever said 'That's not my job'.
+- I can help with literally anything involving writing, editing or proof-reading.
+- I enjoy helping others. You can ask me anything. There are no stupid questions.  
+
+### How to work with me
+- I default to listening. You may need to ask for my thoughts if you want them.
+- I have a very hands-on attitude and will never say 'That's not my job'. 
+- I may parrot ('_It sounds like..._') to ensure I understand. If I'm wrong, tell me. 
+- I'm truly uncomfortable with being in the limelight. I prefer private thank yous.
+- I value compassion and inclusivity above everything else. I try to put people first. 
+- Ad-hoc meetings are fine, but weekends are sacred. I generally work 9-6 GMT.
+
+## Current areas of focus
+### Execution - Q1 2022
+- [ ] **Community:** Organize the first PostHog community meet-up, with a good speaker. Retro the event and roll changes into a second event. Continue to attend and report back on other community events and meet-ups.
+- [ ] **Case studies:** Secure an enterprise-level case study. Enhance the Customers section of the site and make plans to better leverage this content.
+- [ ] **Content:** Support the Content team in creating a decent pipeline of good content that continues to deliver.
+- [ ] **Partnerships:** Find further partnership opportunities to grow the community.
+- [ ] **Internal:** Continue to advocate for less technical team members and users. Become more ambitious when it comes to shipping.
+
+### Personal - Q1 2022
+- [ ] Don't lose (another, again) mission in Pandemic Legacy: Season Zero.
+- [ ] See all of my nieces at least once. And my nephew.
+- [ ] Don't miss a Donut chat.
+
+## Feedback
+In our last [360 feedback session](/handbook/people/feedback#full-team-feedback-sessions-1), I received feedback on the following points. 
+
+- I am seen as an effective and flexible executor, planner, writer and storyteller.
+- I am seen as empathetic and extremely supportive of colleagues, especially new colleagues or those with less experience than me. 
```

Comment:
```suggestion
- I am seen as empathetic and extremely supportive of others, especially new colleagues or those with less experience than me. 
```

Reusing the same word that close together, Joe? Really? I THOUGHT BETTER OF YOU.